packages/@okta/vuepress-site/docs/guides/mobile-idx-sdk-overview/main/index.md

```diff
@@ -39,9 +39,9 @@ Each sign-in step may include one or more possible user actions, such as:
 
 The SDK represents the sign-in flow using a number of different objects:
 
-<div class="common-image-format">
+<div class="common-image-format bordered-image">
 
-![](/img/mobile-sdk/mobile-idx-basic-objects.png "A diagram that shows the SDK objects for the sign in flow and the relationships between them.")
+!["A diagram that shows the SDK objects for the sign in flow and the relationships between them."](/img/mobile-sdk/mobile-idx-basic-objects.png)
```

Comment:
```suggestion
!["A diagram that shows the SDK objects for the sign-in flow and the relationships between them."](/img/mobile-sdk/mobile-idx-basic-objects.png)
```
What type of diagramming standard is this? It's kind of confusing to have Client and Configuration objects with no relationships.

Reply:
fixed